src/Controller/SendSlackEmailController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Message\SendSlackEmail;
+use Psr\Log\LoggerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+/**
+ * Api endpoint for the front end to send message via email to a slack channel
+ */
+class SendSlackEmailController extends AbstractController
+{
+
+    #[Route("/send-slack-email", name: 'send_slack_email')]
+    public function send(Request $request,
+                         MailerInterface $mailer,
+                         LoggerInterface $logger,
+                         ValidatorInterface $validator): Response
+    {
+        $message = $request->request->get('message');
+
+        $emailAddress = $request->request->get('email');
```

Comment:
I like to put the business / validation logic within the controller, if the logic is too complex, I can move it a separate class.